tools/wake-unit/cli_options.cpp

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2023 SiFive, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You should have received a copy of LICENSE.Apache2 along with
+ * this software. If not, you may obtain a copy at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "../tools/wake/cli_options.h"
```

Comment:
I know I said this was ok...but I also remember us realizing that we didn't need it for something. I don't see a way around this here though so it must have been something different. Can you jog my memory?

Reply:
That was for the config test, and basically we landed on integration tests being better there. I could see and accept the same argument here if we just update `wake --config` to also  print out all the flag state.  Would be  a lot noisier though

Reply:
I think Unit tests make a lot of sense for this personally.